ompi/mca/topo/treematch/treematch/tm_kpartitioning.c

```diff
@@ -651,7 +651,6 @@ void free_tab_com_mat(com_mat_t **mat,int k)
       FREE( mat[i]->comm[j] );
     FREE( mat[i]->comm );
     FREE(mat[i]);
-
```

Comment:
We pull tree match from their devel tree, we should try to minimize the non-critical changes.